tests/test_basic.py

```diff
@@ -29,8 +30,8 @@ def test_update_normal():
     assert round(corr_list["xx1"][1], 1) == -0.5
     assert round(corr_list["xx1"][2], 1) == 0.3
 
-## Test handling of variable lists
-def test_predictor_normal_handling():
+## Test handling of variable lists              ###??? What does it mean by 'variable lists'?
```

Comment:
please leave the comment in review section in case we won't lose these comments later on.

Reply:
Yea will do!